pallets/asset-registry/src/lib.rs

```diff
@@ -188,31 +166,14 @@ pub mod pallet {
         pub fn register_asset(
             origin: OriginFor<T>,
             asset: T::AssetType,
-            asset_id: Option<T::AssetId>,
-            metadata: Option<T::AssetRegistrarMetadata>,
-            min_amount: T::Balance,
-            is_sufficient: bool,
+            asset_id: T::AssetId,
         ) -> DispatchResult {
             T::AssetModifierOrigin::ensure_origin(origin)?;
```

Comment:
AssetModifierOrigin can be simplified to `UpdateOrigin`

Reply:
[f858a76](https://github.com/parallel-finance/parallel/pull/1581/commits/f858a76188c11ec088e351ed3876845c45b66614)